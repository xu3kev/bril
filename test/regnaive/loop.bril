main {
  inc: int = const 1;
  length: int = const 10;
  arr1: int = const 5;
  arr2: int = const 1;
  i: int = const 0;
  cond: bool = gt length i;
  br cond loop end;
loop:
  arr3: int = add arr1 arr2;
  i: int = add i inc;
  cond: bool = gt length i;
  br cond loop end;
end:
  print arr3;
}
