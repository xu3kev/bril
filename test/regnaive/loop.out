6
#load: 21
#store: 2
